docs/reference/configuration/observations.rst

```diff
@@ -258,8 +256,27 @@ field, one would add the following to the observation configuration:
 
  HISTORY_OBSERVATION GOPR:FIELD;
 
-By default, the observation error is set to 10% of the observed value,
-with a minimum of 0.10. It can be changed as follows:
+The item ERROR_MODE can take three different values: ABS, REL or RELMIN.
```

Comment:
here you are referring to ERROR_MODE but it is not yet defined. Maybe pull the example 20 lines below up to here in order to introduce the item ERROR_MODE?

Reply:
You can change the configuration example to what is the default setting, that might yield better flow in the explanation.

Reply:
Makes sense :-)